lib/backpex/live_resource.ex

```diff
@@ -1364,28 +1364,27 @@ defmodule Backpex.LiveResource do
       end
 
       @impl Phoenix.LiveView
-      def handle_info({"backpex:" <> unquote(event_prefix) <> "deleted", item}, socket) do
-        if Enum.filter(socket.assigns.items, &(&1.id == item.id)) != [] do
-          {:noreply, refresh_items(socket)}
-        else
-          {:noreply, socket}
-        end
-      end
-
-      @impl Phoenix.LiveView
-      def handle_info({"backpex:" <> unquote(event_prefix) <> "created", item}, socket) do
+      def handle_info({"backpex:" <> unquote(event_prefix) <> "created", item}, socket)
+          when socket.assigns.live_action in [:index, :resource_action] do
```

Comment:
Do we need fallbacks when this (and the other) guard does not match? Or is this not a valid code path?

Reply:
We already have a fallback `handle_info` function added by the  `__before_compile__ ` hook (see [live_resource.ex#L1509](https://github.com/naymspace/backpex/blob/develop/lib/backpex/live_resource.ex#L1509))